src/layer/VideoOverlay.js

```diff
@@ -78,6 +78,7 @@ export var VideoOverlay = ImageOverlay.extend({
 		vid.autoplay = !!this.options.autoplay;
 		vid.loop = !!this.options.loop;
 		vid.muted = !!this.options.muted;
+		vid.playsInline = true;

```

Comment:
Let's make this an option like all the other ones, there might be use-cases where someone would actually want to have this play in full-screen.